### DISCUSSION TITLE
Use ";" as a column separator instead of ","

### DISCUSSION THREAD

**Discussion by sh4rivanoficial** [vzhd1701/csv2notion#40] [Q&A]
Hello! In my country the comma is the decimal separator. Therefore, all of our databases export information using ";" as a separator. The comma is used for values, such as: R$50.00. How can I change the separator that the tool considers?

**Comment by IAmVigneswaran** [comment]
@sh4rivanoficial 

Our latest 1.3.3 version for CSV2Notion Neo has a new `--delimiter` option. 

https://github.com/TheAcharya/csv2notion-neo/releases/tag/v1.3.3
